subtests/docker_cli/systemd/systemd.py

```diff
@@ -5,58 +5,51 @@
 
 Operational Summary
 ---------------------
-This is to test container as systemd service using a predefined unit file
-(``p4321.service``) and a ``script p4321-server.py``. An image is built with
-this script added. Systemd starts a container using this image & the container
-writes current time to port ``4321``. From host, the socket is read and the
-value is checked to see if it is a digit. Finally, container is stopped
-through systemd and system is cleaned up.
+This is to test container as systemd service using a predefined unit file.
+Pulling, building, and running are performed.
 
```

Comment:
Should probably add a ``Prerequisites``  section added here mentioning something about any configured ``image_name`` value and ``Dockerfile`` base image must already exist or be retrievable.